addons/mail/static/src/new/web/chatter.xml

```diff
@@ -16,7 +16,7 @@
                     <span>Activities</span>
                 </button>
                 <span class="o-mail-chatter-topbar-grow flex-grow-1 border-start pe-2" t-att-class="{ 'ms-2': props.hasActivity }"/>
-                <FileUploader t-if="attachments.length === 0" multiUpload="true" onUploaded.bind="onUploaded">
+                <FileUploader t-if="attachments.length === 0" fileUploadClass="'attachment-box-uploader'" multiUpload="true" onUploaded.bind="onUploaded">
```

Comment:
`attachment-box-uploader` => `o-mail-chatter-file-uploader`

Also the file uploader in attachment box should have same classame, as it's the same concept. It's just hidden in a toggle UI element